common4j/src/main/com/microsoft/identity/common/java/opentelemetry/OTelUtility.java

```diff
@@ -80,13 +114,13 @@ public static Attributes getCurrentSpanAttributes() {
     /**
      * Get name of the current span, if possible.
      **/
-    @Nullable
+    @NonNull
     public static String getCurrentSpanName() {
         final Span span = SpanExtension.current();
         if (span instanceof ReadableSpan) {
             return ((ReadableSpan) span).getName();
         }
-        return null;
+        return "null";
```

Comment:
can we just make this an empty string instead of "null"? Was thinking if it may reduce size of a row if we leave it empty.  Although I believe span name is mostly set when we create a span and this is just an edge case. Correct me if I am wrong.

Reply:
Great point. I'll change it to empty string

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1926/commits/9d5bb835341070a37807e077cbd8d4d631582efa